Hamo test

mhadziabdic opened the pull request:
Boki, I fixed the problem that appeared in some computations when wall function is used. I also changed the implementation of wall function so the roughness model is included in more elegant way. I tested this version for some basic cases but it need to be further tested before accepting it. I would like you to look into it and say your opinion on new way roughness is tackled. Zivio.

Niceno commented:
Hamo, this looks good to me, but I find the name suffix "_f" a bit confusing, it suggest to me that the variable refers to a face-based value.  It is sufficient to have just "Turb % z_o(:)" as array, since other "z_o" is a local variable.  I want to say that in each function you can have local "z_o" and variable roughness coefficient in "Turb % z_o(:)".  Unless you fiercely oppose, I would accept this pull request as it is for this branch, but make the above change in the "petsc" branch.

mhadziabdic commented:
I agree, lets keep just z_o. z_o can be allocated just in boundary cells so
we can save some memory.

Regards,

Hamo

Assoc. Prof. Muhamed Hadziabdic
Faculty of Engineering and Natural Sciences
International University of Sarajevo
Hrasnicka cesta 15, 71210, Ilidza, Sarajevo

mob. +387 61 228 489
mail: ***@***.***


On Tue, Jun 14, 2022 at 6:07 AM Bojan Niceno ***@***.***>
wrote:

> Hamo, this looks good to me, but I find the name suffix "_f" a bit
> confusing, it suggest to me that the variable refers to a face-based value.
> It is sufficient to have just "Turb % z_o(:)" as array, since other "z_o"
> is a local variable. I want to say that in each function you can have local
> "z_o" and variable roughness coefficient in "Turb % z_o(:)". Unless you
> fiercely oppose, I would accept this pull request as it is for this branch,
> but make the above change in the "petsc" branch.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/DelNov/T-Flows/pull/241#issuecomment-1154685637>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHRRJ6WZUEMOMMJXPK7GFCLVPAAQDANCNFSM5YOKHQFA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

mhadziabdic commented:
Boki I added two more changes which I somehow missed to add in the first pull request. I mentioned about it. I moved calculation of turbulent scalar flux from Compute_Scalar.f90 to Turb_Mod/Main.f90.
